apps/courses/src/routes/courses/search/index.tsx

```diff
@@ -75,13 +68,13 @@ export default () => {
 
     return hasSkillLevel && hasTopic && hasLanguages;
   };
+  
+  const filterData = (data) =>
```

Comment:
If this is the only time you use this function, just perform the operation inside of `useEffect` and don't define it as a variable.

Reply:
Replace fuse in a long run